subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/EmulatorsLogsReporterImpl.kt

```diff
@@ -10,10 +10,9 @@ internal class EmulatorsLogsReporterImpl(
 ) : EmulatorsLogsReporter {
 
     override fun reportEmulatorLogs(emulatorName: Serial, log: String) {
-        getFile(
-            dir = File(outputFolder, DEVICES_LOGS),
-            emulatorName = emulatorName.value
-        ).writeText(log)
+        getLogFile(emulatorName.value)
+            .apply { parentFile?.mkdirs() }
+            .writeText(log)
```

Comment:
Maybe append?
If I'm not mistaken there are no restrictions to get a new restarted pos with IP from previously destroyed.

Reply:
ye, you probably right, and this is a mistake